frappe/www/login.py

```diff
@@ -41,6 +41,7 @@ def get_context(context):
 	context["title"] = "Login"
 	context["provider_logins"] = []
 	context["disable_signup"] = frappe.utils.cint(frappe.get_website_settings("disable_signup"))
+	context["disable_user_pass_login"] = frappe.utils.cint(frappe.get_system_settings("disable_user_pass_login"))
```

Comment:
Based on first impression it looks like this checkbox just removes user/pass section from login page. However, one can very easily log in by sending a POST request. We should disable that too with same settings. 



```javascript
fetch("http://site", {
  "headers": {
    "accept": "application/json, text/javascript, */*; q=0.01",
    "accept-language": "en-GB,en;q=0.9",
    "content-type": "application/x-www-form-urlencoded; charset=UTF-8",
    "x-frappe-cmd": "login",
    "x-requested-with": "XMLHttpRequest"
  },
  "body": "cmd=login&usr=Administrator&pwd=admin&device=desktop",
  "method": "POST",

});
```

Reply:
API keys are different so they are fine 👍 


API logins using user/pass should be disabled too. You'll find this login in `auth.py`